src/edu/ucsb/cs56/projects/games/fish_animation/FishAnimationEnvironment.java

```diff
@@ -303,6 +305,10 @@ public void paintComponent(Graphics g) {
 				if (boatX <= -250) {
 					boatX = this.getWidth();
 				}
+				if ((boatX - 20 > posX - 200 && boatX + 20 < posX + 40) && (-135 + 60 > posY - 60 && -135 < posY + 25)) {
```

Comment:
I don't think this should be part of this PR.  Isn't there a different one for it?

Reply:
Just removed it